### PR TITLE
Create Tweet: 221106-classic-memes.tweet

### PR DESCRIPTION
**Pull request by IstoraMandiri**
Approve, don't merge. Thanks.

### CODE REVIEW

**Review by gitr0n1n — APPROVED** [review]

**Comment by w1g0** [comment]
@gitr0n1n as ETC_Network only need one approval, why did you request more from @ethereumclassic/tweeters?
This is on hold for merge on @IstoraMandiri's request.

**Pull request merged**
